diff -Nur -x '*.orig' -x '*.rej' e-smith-lib-compspec-0.3.0/root/etc/profile.d/e-smith-lib_compspec.sh mezzanine_patched_e-smith-lib-compspec-0.3.0/root/etc/profile.d/e-smith-lib_compspec.sh
--- e-smith-lib-compspec-0.3.0/root/etc/profile.d/e-smith-lib_compspec.sh	2005-07-16 11:42:30.264776768 -0600
+++ mezzanine_patched_e-smith-lib-compspec-0.3.0/root/etc/profile.d/e-smith-lib_compspec.sh	2005-07-16 11:41:46.364171376 -0600
@@ -7,7 +7,7 @@
     1)
     	COMPREPLY=( $(find /etc/e-smith/events/ -type d \
 		      \( -name actions -prune -o \
-		         -name '$cur*' -printf "%f\n" \)) )
+		         -name "$cur*" -printf "%f\n" \)) )
 	;;
     *) 	;;
     esac
@@ -46,7 +46,7 @@
     1) # config file
 	COMPREPLY=( $(find /home/e-smith -maxdepth 1 -type f \
 			\( -name '.*' -prune -o \
-			   -name '$cur*' -printf "%f\n" \)) )
+			   -name "$cur*" -printf "%f\n" \)) )
 	;;
     2) # subcommand 
 	COMPREPLY=( $(/sbin/e-smith/db 2>&1 |awk '{print $3}' \
